tests/e2e/visual/targets/targets-overview.wdio-spec.js

```diff
@@ -0,0 +1,61 @@
+const utils = require('@utils');
+
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const pregnancyVisitFactory = require('@factories/cht/reports/pregnancy-visit');
```

Comment:
Can you please also create pregnancy visits reports? So, we have values in routine facility visit targets.